applications/crossbar/src/modules/cb_ledgers.erl

```diff
@@ -187,6 +180,19 @@ validate(Context, SourceService) ->
     crossbar_view:load_modb(Context, ?VIEW_BY_SOURCE, ViewOptions).
 
 -spec validate(cb_context:context(), path_token(), path_token()) -> cb_context:context().
+validate(Context, ?SUMMARY, ModbSuffix) ->
+    case ?CURRENT_MODB of
```

Comment:
What if I want to get the summary about past months? pattern match to check if `ModbSuffix` is in format of YYYYMM and use `kazoo_mdb:get_modb(cb_context:account_id(Context), Year, Month)` to get the correct modb.